### ISSUE TITLE
Pretty difficult to avoid, but the framing on this conflates BTC addresses with accounts

### ISSUE THREAD

**Issue by emansipater** [DZGoldman/awdy#4]
Since the metric seems to be aiming for *entities* rather than "clumps" it really should be possible to track, i.e. Satoshi's known reserves within this metric. Would definitely take a good bit of work to make this a live statistic though.

**Comment by DZGoldman** [comment]
Indeed, it's a very narrow, easily game-able metric; my compromise is to just make explicit that it's measuring "addresses." 
The problem is even if we can clump some of the bigger entities for, say, Bitcoin and Ethereum, I imagine it will be very hard to find such data for smaller coins, and applying different analyses seems unfair. Open to suggestions tho
